packages/sdk/src/oracle/services/statemachines/setActiveRequest.ts

```diff
@@ -0,0 +1,38 @@
+import { Update } from "../update";
+import Store from "../../store";
+import { Inputs } from "../../types/state";
+import { Handlers as GenericHandlers } from "../../types/statemachine";
+import { getFlags } from "../../utils";
+
+// required exports for state machine
+export type Params = Inputs["request"];
+export type Memory = { error?: Error };
+export function initMemory(): Memory {
+  return {};
+}
+export function Handlers(store: Store): GenericHandlers<Params, Memory> {
```

Comment:
this is the statemachine handler for the setActiveRequest call. This does not need any state transitions, so technically this is more of a proof of concept for more complex state machines. These files require certain exports to be able to be imported into the state machine: `Params`, `Memory`, `Handlers` and `initMemory`. Tried to make this as easy given the contstraints of typescript. All future context handlers should fallow this pattern.